src/m_menu.c

```diff
@@ -3953,33 +3961,39 @@ setup_menu_t gen_settings2[] = { // General Settings screen2
 
 setup_menu_t gen_settings3[] = { // General Settings screen3
 
-  {"Miscellaneous"  ,S_SKIP|S_TITLE, m_null, M_X, M_Y},
+  {"Quality of life"  ,S_SKIP|S_TITLE, m_null, M_X, M_Y},
 
   {"Strict Mode", S_YESNO|S_LEVWARN, m_null, M_X,
    M_Y + general_strictmode*M_SPC, {"strictmode"}},
 
+  {"Show demo progress bar", S_YESNO, m_null, M_X,
+   M_Y + general_demobar*M_SPC, {"demobar"}},
+
+  {"On death action", S_CHOICE, m_null, M_X,
+   M_Y + general_death_action*M_SPC, {"death_use_action"}, 0, NULL, death_use_action_strings},
+
+  {"Palette changes", S_YESNO, m_null, M_X,
```

Comment:
Isn't this rather the first step towards Accessibility (although "Quality of Life" sounds fine as well).

Reply:
There is only two A11y options so far (maybe three, with bobbing). I think we can add another menu group if we implement more.

Reply:
Yes, right!